Dangerfile

```diff
@@ -0,0 +1,28 @@
+# MessageKit, 2020
+
+# This runs on CI
+mergeable_state = github.pr_json["mergeable_state"]
+
+# Make it more obvious that a PR a draft
+if mergeable_state == "draft"
+  warn("PR is marked as Draft")
+end
+
+# Mainly to encourage writing up some reasoning about the PR, rather than just leaving a title
+if github.pr_body.length < 5
+  fail("Please provide a summary in the Pull Request description")
+end
+
+declared_hashtag = github.pr_title.include?("#trivial")
+if !git.modified_files.include?("CHANGELOG.md") && !declared_hashtag
```

Comment:
If you're going to move one condition out of the in-line, maybe they both should be assigned to a variable for readability & consistency:
```
has_trivial_hashtag = github.pr_title.include?("#trivial")
has_changelog = git.modified_files.include?("CHANGELOG.md")
if !has_trivial_hashtag && !declared_hashtag
```